content/references/data-api.md

```diff
@@ -3088,19 +3055,17 @@ A successful response uses the HTTP code **200 OK** and has a JSON body with the
 
 Daily Validator Report Object fields:
 
-| Field                   | Value                           | Description      |
```

Comment:
SUPER nit about the intro sentence above. Considering changing "Get a validation vote stats" to "Get validation vote stats"?